master/internal/trial_test.go

```diff
@@ -1,3 +1,4 @@
+//nolint:exhaustivestruct
```

Comment:
Just curious how much due diligence was done on these. Are these all experiment configs like the one above, or are these ignored just when they're in tests because that's less critical?

Reply:
so this is a test, so I wasn't super concerned, tests make weird partial versions of stuff all the time (but i did look at all of them). any time it didn't appear in a test i read carefully and left a comment about why it was ok.